tests/model/templates/tester-response-template.model.spec.ts

```diff
@@ -14,4 +26,22 @@ describe('TesterResponseTemplate class', () => {
       expect(template.teamChosenType).toBe(TYPE_DOCUMENTATION_BUG);
     });
   });
+  describe('testerResponse and teamResponse fields', () => {
+    it('parses the testerResponse and teamResponse fields correctly from the GithubComment', () => {
+      const template = new TesterResponseTemplate([TEAM_RESPONSE_MULTIPLE_DISAGREEMENT]);
+      console.log(template.testerResponse.testerResponses[1].disagreeCheckbox);
```

Comment:
`console.log` statement missed out here!

Reply:
Thanks for spotting this! Updated the PR accordingly 😄